content/posts/web-components.md

```diff
@@ -125,28 +106,32 @@ correct.
 
 > Annotations shown in the Verification above were sourced from the
 > [Cooloola Ecosounds project](https://www.ecosounds.org/projects/1029)
-> under the Creative Commons by Attribution 4.0 license.
-
-> Phillips, Yvonne (2018): Environmental long-duration audio recordings: Gympie and Woondum National Park . Queensland University of Technology. (Sound) <https://doi.org/10.4225/09/5a7297ee4b893>
-
-While it is recommended to use the Ecosounds or A2O verification workflow, the
-nature of these web components being portable means that they are fully
-customizable and can be embedded for any task outside of Ecosounds or the A2O
-and supports a wide variety of data formats.
+> under the Creative Commons by Attribution 4.0 license from 
+> Phillips, Yvonne (2018) <https://doi.org/10.4225/09/5a7297ee4b893>
 
 Creating your own verification grid also allows easy customization of the
 verification task. For example, if your dataset does not include labeled
-species, we support tagging audio through the
+species, we support labelling audio segments through the
 [classification](https://oe-web-components.netlify.app/examples/verification/classification/)
-variant of the `oe-verification-grid` web component.
+variant of the `oe-verification-grid` web component. See the
+[Verification Grid Example](https://stackblitz.com/edit/oe-web-components-basic?file=examples%2Fverification-grid.html).
+
+## Ecosounds and A2O integration
+
+We are using the spectrogram component in some parts of Ecosounds and the A2O already.
```

Comment:
```suggestion
We are already using the spectrogram component in some parts of Ecosounds and the A2O.
```

This sentence still doesn't read well to me, but I think moving "already" from the end of the sentence sounds a lot better.